src/Scores.cpp

```diff
@@ -154,6 +161,93 @@ void ScoreHolder::printPeptide(ostream& os, bool printDecoys, bool printExpMass,
   }
 }
 
+std::string getCharge(string id) {
```

Comment:
Is this realy meant as a free floating function? Could yo make it a member function of e.g. Scores?